posthog/models/feature_flag.py

```diff
@@ -333,3 +421,26 @@ def get_overridden_feature_flags(
             feature_flags[key] = value
 
     return feature_flags
+
+
+def set_feature_flag_hash_key_overrides(
+    feature_flags: QuerySet, team_id: int, person_id: int, hash_key_override: str
+) -> None:
+
+    existing_flag_overrides = set(
+        val.feature_flag_key
+        for val in FeatureFlagHashKeyOverride.objects.filter(team_id=team_id, person_id=person_id).only(
```

Comment:
You can use `.values_list("feature_flag_key", flat=True)` and remove the list comprehension